docs/cudf/source/user_guide/data-types.md

```diff
@@ -136,7 +136,7 @@ dtype: struct
 StructDtype({'a': dtype('int64'), 'b': dtype('int64')})
 ```
 
-Or by reading them from disk, using a [file format that supports nested data](io).
+Or by reading them from disk, using a [file format that supports nested data](user_guide/io/index).
```

Comment:
This change didn't work. I thought it would fix the problem but it raises a different error. Feel free to make a suggestion if you know how to resolve this, and I'll try again soon as well.

```
/__w/cudf/cudf/docs/cudf/source/user_guide/data-types.md:139: WARNING: 'myst' cross-reference target not found: 'user_guide/io/index' [myst.xref_missing]
```

Reply:
I discussed a bit with @wence- and found a working solution in f3cfbdf, but it's still unclear to me how this is supposed to work. I read the docs (https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#default-destination-resolution) and it helped answer some questions but there are still alternatives that I thought should work, like `#user_guide/io/index` to trigger part 3(iii), an internal reference to a target elsewhere in the project. 🤷‍♂️